src/compiler/post-transform/define-meta/insert-parameters.ts

```diff
@@ -68,6 +76,34 @@ export function insertDefineMetaJSDocCommentAsDescription(params: Params): void
     );
   }
 
+  // Related to case 2
+  const disableSaveFromUIIndex = findPropertyDisableFromUIIndex({
+    filename,
+    node: defineMetaFirstArgumentObjectExpression,
+  });
+  // Related to case 2
+  if (disableSaveFromUIIndex === -1) {
+    getParametersPropertyValue({
+      filename,
+      node: defineMetaFirstArgumentObjectExpression,
+    }).properties.push(
+      createASTProperty('disableSaveFromUI', {
+        type: 'Literal',
+        value: true,
+      })
+    );
+  } else {
+    // WARN: We're overriding the existing value for user
```

Comment:
I'm not sure if this was the right decision. My gut tells me that we _should_ override for the user, because this feature will not work for the Svelte CSF addon, yet.